CHANGELOG.md

```diff
@@ -268,72 +268,47 @@
 
   ([PR #1376](https://github.com/alphagov/govuk-frontend/pull/1376))
 
-- Ensure GOV.UK Frontend component selectors cannot conflict when initialised
+- Update and add the data-module attribute
 
-  Components which have JavaScript functionality include a data attribute called `[data-module]`,
-  we have added the `govuk-` prefix as a namespace.
+  The `[data-module]` attribute in components now has a `gov-uk` namespace prefix. This is to avoid conflicts between attribute names, and make the attributes consistent with our style for class names.
 
-  This ensures the component's name does not conflict with other service level components
-  with the same name, which is consistent with our CSS class name convention.
+  The [button](https://design-system.service.gov.uk/components/button/#start-buttons) and [details](https://design-system.service.gov.uk/components/details/) components now also have the `[data-module]` attribute.
 
-  If you're using Nunjucks macros, and are [using the `initAll` function](https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/installing-with-npm.md#option-1-include-javascript) you will not be affected.
+  You'll need to update your CSS or JavaScript code, to make sure that JavaScript can find and initalise the components.
 
-  If you are using HTML, you will need to add the `govuk-` prefix to any `[data-module]` attributes.
+  You do not need to do anything if you're using Nunjucks macros and the [`initAll`](https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/installing-with-npm.md#option-1-include-javascript) function.
 
-  For example the accordion component selector has changed from `[data-module="accordion"]` to `[data-module="govuk-accordion"]`.
+  -- If you're using HTML and CSS
+
+  You'll need to add the `govuk-` prefix to the `[data-module]` attribute in each CSS selector.
+
+  For example:
 
   ```html
   <div class="govuk-accordion" data-module="govuk-accordion">
-    <!-- ... -->
+  ...
   </div>
   ```
 
-  If you are initialising components manually, you will need to add the `govuk-` prefix to any CSS selectors used to find GOV.UK Frontend components:
+  You'll also need to add `data-module="govuk-button"` to each:
```

Comment:
This is incorrect since we don't want to add 'govuk-button' to the details component